Zappy_IA_Report/DTC/Code/zappyIa.py

```
@@ -0,0 +1,20 @@
+
+import process
+import game
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2 and len(sys.argv) != 5 | len(sys.argv) != 7:
+        sys.stderr.write("Invalid number of arguments (-help for more informations).\n")
+        sys.exit(84)
+
+    if len(sys.argv) == 2:
+        if sys.argv[1] == "-help":
+            info.info()
```

Ruff (0.11.9)

Undefined name `info`

(F821)